changelog/3367.bugfix.rst

```diff
@@ -0,0 +1 @@
+Updating both GOES XRS and LYRA dataretriver clients to use Scraper
```

Comment:
Is this actually fixing a bug? Does it have any external facing changes at all?

Reply:
I mean it might not be user facing but the change could in theory fix something we've overlooked.

Reply:
it doesn't have external facing changes - other than before the Fido search would have returned a query result that may not actually exist so it fails on download e.g. #3349 and #3337

Reply:
If it's not actually fixing a bug I would probably classify it as "internal". I have a feeling it might be fixing bugs where some files were missing on the server though. If it is a bug fix I think the changelog entry should describe the bug it's fixing.

Reply:
It returning results that aren't on the server counts as a bug in my mind, can you edit the changelog to describe that @hayesla ? :smile: